notebooks/en/rag_with_knowledge_graphs_neo4j.ipynb

```diff
@@ -0,0 +1,644 @@
+{
```

Comment:
<p>"Langchain RetrievalWA creates a question-answering (QA) chain using the above vector index as a retriever."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>